src/Elm/Parser/State.elm

```diff
@@ -35,7 +35,7 @@ expectedColumn (State { indents }) =
 
 pushIndent : Int -> State -> State
 pushIndent col (State s) =
-    State { s | indents = (col + 1) :: s.indents }
+    State { s | indents = (col - 1) :: s.indents }
```

Comment:
Motivating example:
```elm
case f of
  True -> 1
  False -> 2
```
The case..of parser was at the letter `T`, with location `{row = 2, column = 3 }`.
You can see there is indentation of 2 spaces in the snippet.
Before the fix, `pushIndent` was reading that as indentation of 4 spaces instead of 2.